app/src/main/java/com/gh4a/activities/PullRequestActivity.java

```diff
@@ -614,4 +711,43 @@ protected String getErrorMessage() {
             return getContext().getString(R.string.pull_error_merge, mPullRequest.getNumber());
         }
     }
+
+
+    private class PullRequestReviewTask extends ProgressDialogTask<Review> {
+        private final String mReviewMessage;
+        private final String mReviewEvent;
+
+        public PullRequestReviewTask(String reviewMessage, String reviewEvent) {
+            super(getBaseActivity(), R.string.reviewing_msg);
+            mReviewMessage = reviewMessage;
+            mReviewEvent = reviewEvent;
+        }
+
+        @Override
+        protected ProgressDialogTask<Review> clone() {
+            return new PullRequestReviewTask(mReviewMessage, mReviewEvent);
+        }
+
+        @Override
+        protected Review run() throws Exception {
+            PullRequestService pullService = (PullRequestService)
+                    Gh4Application.get().getService(Gh4Application.PULL_SERVICE);
+            RepositoryId repoId = new RepositoryId(mRepoOwner, mRepoName);
+            mReview = pullService.createPullRequestReview(repoId, mPullRequest.getNumber(), mReviewEvent, mReviewMessage);
+            return mReview;
+        }
+
+        /* There's nothing to be done on success */
+        @Override
+        protected void onSuccess(Review result) {
+            mReview = result;
```

Comment:
We should reload all comments so the newly created review will be added to the screen. Also the `mReview` field appears to be unused so you can remove it.